irohad/main/application.cpp

```diff
@@ -30,7 +30,7 @@ void Irohad::run(){
   iroha::model::ModelCryptoProviderImpl crypto_provider(keypair.privkey, keypair.pubkey);
 
   iroha::validation::StatelessValidatorImpl stateless_validator(crypto_provider);
```

Comment:
Maybe, we potentially haven't taken agreement about interface and impl yet.
I think we have 2 opinions as follows:
One opinion is: every module should have inteface and implementation for a convinient test and clear dependency.
Another opinion is: only some module that has a dependency with external library (grpc) should have interface because stub will be removed after allm it's redundant that all module should have `Impl` suffix, and many files are created and hard to manage them.